gql/inputs/insightInputs/leaveDayRequests.input.go

```diff
@@ -39,6 +39,14 @@ func (ldi *LeaveDayRequestsInput) ToPaginationDataAndQuery() (LeaveDayRequestsQu
 		if ldi.Query.UserIdEq != nil {
 			query.UserIdEq = ldi.Query.UserIdEq
 		}
+
+		if ldi.Query.FromGteq != nil {
+			query.FromGteq = ldi.Query.FromGteq
+		}
+
+		if ldi.Query.ToLteq != nil {
+			query.ToLteq = ldi.Query.ToLteq
+		}
```

Comment:
không nên để như này, vì cái này là bản chất là cần query theo `time` nên cần convert qua time trước khi đưa vào query, nếu convert qua time bị lỗi thì query.From / query.To = nil